assets/src/scripts/components/hydrograph/index.spec.js

```diff
@@ -150,7 +149,7 @@ const TEST_STATE = {
 };
 
 
-describe('Hydrograph charting module', () => {
+describe('Loading indicators and data alerts', () => {
```

Comment:
The remaining tests might belong in the time-series-graph.spec, but the loading indicators seem to be used in other areas of the page so I left them here for the time being.

Reply:
Yes, there are actually two loading indicators on the page so it's appropriate to leave this here